Makefile

```diff
@@ -8,11 +8,16 @@ HELIO_OPENSSL_USE_STATIC_LIBS = ON
 HELIO_ENABLE_GIT_VERSION = ON
 HELIO_WITH_UNWIND = OFF
 
+# equivalent to: if $(uname_m) == x86_64 || $(uname_m) == amd64
+ifneq (, $(filter $(BUILD_ARCH),x86_64 amd64))
+HELIO_MARCH_OPT := -march=core2 -msse4.1 -mpopcnt -mtune=skylake
```

Comment:
I think `skylake` is a mistake here. They are running cpu family 6 model 13 which according to [this](https://en.wikichip.org/wiki/intel/cpuid) is a `pentium m`. So you are tuning for an architecture that came later. Furthermore, as per the comments, shouldn't it be `-march=native`? 

What am I missing here?

Reply:
Our responsibility as project maintainers to do what's right and not what was asked in the comments :)

We lowered the bar but we are not going to support 32bit processor pentium-m discontinued in 2009. 
`march=native` will produce a code based on Azure github runner CPU. That's totally random and will be incompatible with many machines that run Dragonfly today. I will update the commit description

Reply:
+1 I got confused because you wrote `Possibly addresses` and the changes introduced in this PR should be incompatible with what is discussed on that thread.

Reply:
Yes, fixed the description.